tests/clean.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+tests="$tests $(find . -maxdepth 1 -type d -name 'test_*' | sort)"
+
+count=$(echo $tests | wc -w)
+
+# Print number of tests detected
+echo "1..$count tests"
+
+for t in $tests ; do
+    if [ -e $t/clean.sh ] ; then
+        echo $t OK
+        bash -c -l "cd $t && ./clean.sh"
+    else
+        echo $t have not clean.sh script
```

Comment:
"have no clean.sh" -> "does not have a clean.sh"

Reply:
Thanks.